dams-tests/tests/common.rs

```
@@ -129,7 +126,8 @@ async fn client_test_config() -> dams::config::client::Config {
 
 /// Encode the customizable fields of the keymgmt server Config struct for
 /// testing.
-async fn server_test_config() -> dams::config::server::Config {
+pub async fn server_test_config() -> dams::config::server::Config {
+    let _ = fs::create_dir("tests/gen");
```

I think it's ok to `.expect` here since it's in the tests. If there's a problem creating the directory this will swallow up the error and we won't know about it until later when the directory is expected to exist.